src/components/shared/articleBody.tsx

```diff
@@ -90,7 +90,7 @@ const ArticleBody = ({
     imageSalt,
     className,
 }: ArticleBodyProps): JSX.Element =>
-    <div css={[className, ArticleBodyStyles(pillarStyles), ArticleBodyDarkStyles(pillarStyles)]}>
+    <div css={[className, ArticleBodyStyles(pillarStyles), ArticleBodyDarkStyles(pillarStyles)]} className="article__body">
```

Comment:
This `className` appears a bit mysterious to a newcomer. As far as I can tell it comes from the styles in `ImmersiveArticleBody`, but relies on this component being imported from there and won't be applied otherwise? Is it possible to pass styles into the component from there instead?